### DISCUSSION TITLE
Native App with UI for Windows/Mac/Linux

### DISCUSSION THREAD

**Discussion by julien-boudry** [julien-boudry/Condorcet#170] [Ideas]
Could use NativePHP framework (electron based) + Laravel.

**Comment by LiamM32** [comment]
Alright. I suppose this would make Condorcet accessible for those who don't know how to use the terminal. I don't know if there's many potential users of this software who don't know how to use the terminal, but there might be some things that can be made easier to do with a GUI than when inputting votes through text.

I don't know anything about GUI applications in PHP. The only major GUI toolkit I used was Qt with C++, though I am now a D programmer, so I would more likely use GTK with D if I were making a new GUI application.
